web/controllers/page_controller.ex

```diff
@@ -1,5 +1,6 @@
 defmodule Booky.PageController do
   use Booky.Web, :controller
+  plug Booky.Plug.AuthenticateUser
```

Comment:
hmm. it doesn't apply to the oauth routes though... I'll see if I can chain pipelines